glib/gmessages.c

```diff
@@ -1947,6 +1947,59 @@ g_log_structured_array (GLogLevelFlags   log_level,
     _g_log_abort (!(log_level & G_LOG_FLAG_RECURSION));
 }
 
+/* Semi-private helper function to implement the g_message() (etc.) macros
+ * with support for G_GNUC_PRINTF so that @message_format can be checked
+ * with -Wformat. */
+void
+g_log_structured_standard (const gchar    *log_domain,
```

Comment:
The commit message refers to `_g_log_structured_standard` but that's not the name you used here. I guess it's a bit late to make this point since the patch was already merged upstream.

Reply:
Oops, well spotted. Yes, it’s too late to change that. :'-(